modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,98 @@
+var username = $("#navbar_profile_link").text().substr(1);
+if (document.getElementById("navigation_top")){
+    var Credits = document.getElementById("navigation_top").outerHTML;
+}
+if (document.getElementById("coins_top")){
+    var Coins = document.getElementById("coins_top").outerHTML;
+}
+var credits = document.getElementById("navigation_top");
+var coins = document.getElementById("coins_top");
+var gesamtcredits = "undefined";
+var dienstgrad = "undefined";
+var next_level = "undefined";
+
+function lightbox_open() {
+    lightboxOpen();
+}
+
+function setCookies() {
+  gesamtcredits = NaN;
+  dienstgrad = NaN;
+  content = $(".lightbox_iframe");
+  content = content.first()[0];
+  content = content.contentWindow.document;
+  content = content.children[0].children[1].children[2].children[0];
+  content = content.innerHTML;
+  content = content.replace(/<\/?[^>]+>/gi, "");
+  content = content.replace(/(.|\n)*Dienstgrad: /g,"");
+  content = content.replace(/ verdiente(.|\n)*/g,"");
+  gesamtcredits = content.replace(/(\d|.)*\n../g, "");
+  gesamtcredits = gesamtcredits.replace(/\./g, "");
+  dienstgrad = content.replace(/\n.*/g, "");
+  document.cookie = "gesamtcredits=" + gesamtcredits + "; path=/";
+  document.cookie = "dienstgrad=" + dienstgrad + "; path=/";
+  console.log("Saved Cookies!");
+}
+
+function readCookie(name) {
+    var nameEQ = name + "=";
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') c = c.substring(1,c.length);
+        if (c.indexOf(nameEQ) === 0) return c.substring(nameEQ.length,c.length);
+    }
+    return null;
+}
+
+var dienstgrade = ["Anwärter(in)","Feuerwehrmann / Feuerwehrfrau","Oberfeuerwehrmann / Oberfeuerwehrfrau","Hauptfeuerwehrmann / Hauptfeuerwehrfrau","Stv. Gruppenführer(in)","Gruppenführer(in)","Stv. Zugführer(in)","Zugführer(in)","Stv. Wehrführer(in)","Wehrführer(in)","Stv. Kreisbrandmeister(in)","Kreisbrandmeister(in)","Stv. Landesbrandmeister(in)","Landesbrandmeister(in)","Ehrenmitglied"];
+var benoetigte_credits = [0,200,10000,100000,1000000,5000000,10000000,20000000,50000000,100000000,200000000,500000000,1000000000,2000000000,5000000000];
```

Comment:
Du schreibst Variablen mal englisch und mal deutsch, sowie mal in camelCase und mit unter_strich. Vereinheitliche das bitte.
Am besten englischLowerCamelCase